src/migrations/20220812205424-new-ar-topics.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable max-len */
+
+// An array of objects with the old and new topic names.
+const topicMapping = [
+  { oldName: 'Child Assessment, Development, Screening', newName: 'Child Screening and Assessment' },
+  { oldName: 'Teaching Practices / Teacher-Child Interactions', newName: 'Teaching / Caregiving Practices' },
+];
+
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const loggedUser = '0';
+      const sessionSig = __filename;
+      const auditDescriptor = 'RUN MIGRATIONS';
+      await queryInterface.sequelize.query(
+        `SELECT
+          set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+          set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+          set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+          set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+        { transaction },
+      );
+
+      // Disable audit logging
+      await queryInterface.sequelize.query(
+        `
+          SELECT "ZAFSetTriggerState"(null, null, null, 'DISABLE');
+          `,
+        { transaction },
+      );
```

Comment:
Delete this, so audit log will record it. Per conversation Nathan & Garrett

Reply:
Done: 0b4d25dc27e6f8fafab0fa233e995b3ec4fa4dff